react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -72,7 +73,14 @@ class OverflowMenu extends Component<Props> {
                 <AudioOnlyButton { ...buttonProps } />
                 <RoomLockButton { ...buttonProps } />
                 <ClosedCaptionButton { ...buttonProps } />
-                <RecordButton { ...buttonProps } />
+                {
+
+                    // Apple rejected our app because they claim requiring a
+                    // Dropbox account for recording is not acceptable.
+                    // Ddisable it until we can find a way around it.
+                    Platform.OS !== 'ios'
+                        && <RecordButton { ...buttonProps } />
```

Comment:
I'd be happier if this would be disabled inside the button (render null). You know I'm a big fan of self-aware components ;)

Reply:
Ok, can do.